### PR TITLE
Add Docker-Compose, AWS & Upgrade Node, Java

### LINKED ISSUE #6: Install docker-compose
```
sudo curl -L "https://github.com/docker/compose/releases/download/1.25.4/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
sudo chmod +x /usr/local/bin/docker-compose
```

### LINKED ISSUE #7: Add aws cli
https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-linux.html

### PR DESCRIPTION
**Pull request by jonnyelliot**
- Updates base image to use AdoptOpenJDK 13 Ubuntu
- Adds Docker Compose tool
- Adds AWS CLI tool
- Adds drone build to ease building/testing changes
- Updates Node to latest LTS
- Updates README to include pinned tool versions, adds licence references
- Updates CI tests to include all tools

Closes #6
Closes #7

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+kind: pipeline
+type: docker
+name: committed-ci:1.4.0
```
Comment (stuarthendren) [inline .drone.yml]: Probably doesn't need the version

```suggestion
name: committed-ci
```

File: README.md [hunk 2]
```diff
@@ -20,17 +20,19 @@ The following general libs and utilities are installed:
 
 The build tools included are:
 
-- **Java** openjdk 11.0.2 2019-01-15
```
Comment (stuarthendren) [inline README.md:23]: Why change to latest?

Would soon be wrong

**Review by stuarthendren — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jonnyelliot) [reply README.md:23]: Latest is the wrong word I agree. I just mean that the versions arent pinned (will update whenever the image is rebuilt)

Comment (jonnyelliot) [reply README.md:23]: I was going to update all the versions here but I know they'd become out of date before long

Comment (jonnyelliot) [reply README.md:23]: I've now removed the "latest" wording entirely for unpinned stuff

Comment (stuarthendren) [reply README.md:23]: that seems like the wrong thing to be doing you don't want your tools changing from one ci build to the next.
 
Should fix the base version then they would be pinned.

The tests could be extended to print all these version to make it easier to update.

**Review by stuarthendren — CHANGES_REQUESTED** [review]
Fix the base image  version

**Comment by jonnyelliot** [comment]
@stuarthendren I've fixed the base image version and now list all versions out in the README.

**Review by stuarthendren — APPROVED** [review]

**Pull request merged**